assets/js/pages/ClusterDetails/HanaClusterDetails.test.jsx

```diff
@@ -466,6 +466,53 @@ describe('HanaClusterDetails component', () => {
     }
   );
 
+  it.each([
+    { arch: 'angi', tooltip: 'Angi architecture' },
+    { arch: 'classic', tooltip: 'Classic architecture' },
+  ])(
+    'should show cluster type with $arch architecture',
```

Comment:
why $ in arch ?  :)

Reply:
To interpolate the value of `arch` value coming in the scenarios